.github/workflows/pythonbuild.yml

```diff
@@ -58,6 +56,37 @@ jobs:
       with:
         fail_ci_if_error: true # optional (default = false)
 
+  build-plugins:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+        # Omitting "flytekit-pandera" for now.
+        plugin-names: ["flytekit-aws-athena", "flytekit-aws-sagemaker", "flytekit-data-fsspec", "flytekit-dolt", "flytekit-greatexpectations", "flytekit-hive", "flytekit-k8s-pod", "flytekit-kf-pytorch", "flytekit-kf-tensorflow", "flytekit-papermill", "flytekit-spark", "flytekit-sqlalchemy"]
```

Comment:
is it possible to get this using `ls plugins/flytekit-*` or maybe a file `cat plugins/tests.yaml`

Reply:
can we do this in the future?  i know this isn't pretty.  i can make another ticket for it.